tests/golos.publication_tests.cpp

```diff
@@ -77,6 +78,14 @@ class golos_publication_tester : public golos_tester {
         }
     }
 
+    void prepare_balances(int supply = 1e5, int issue1 = 500, int issue2 = 500, int buy1 = 100, int buy2 = 100) {
+        BOOST_CHECK_EQUAL(success(), token.create(cfg::emission_name, token.make_asset(supply), {cfg::charge_name, N(golos.pub)}));
```

Comment:
As I understand, `cfg::charge_name, N(golos.pub)` not need here. You can use `_code` (account for publication contract).